torch/csrc/PythonTypes.h

```diff
@@ -12,4 +12,9 @@ struct THPVoidTensor {
   char data_type;
 };
 
+struct THPVoidStorage {
+  PyObject_HEAD
+  THVoidStorage *cdata;
+};
```

Comment:
Why do we need this now? Shouldn't we be moving away from these void hacks?

Reply:
We still need a way to go from ATen storage to `THPStorage` for now. I don't think it's worth doing a ATen-based bindings to Python for Storage at the moment.